projects/payment-lib/src/lib/interfaces/NotificationPreviewRequest.ts

```diff
@@ -41,7 +42,9 @@ export class NotificationPreviewRequest {
             ccd_case_number: (payment !== undefined && payment !== null) ? payment.ccd_case_number : '',
             refund_reason: refund_reason,
             refund_amount: refund_amount,
-            refund_reference: refund_reference
+            refund_reference: refund_reference,
+            customer_reference: (payment !== undefined && payment !== null && payment.customer_reference !== undefined
+              && payment.customer_reference !== null) ? payment.customer_reference : ''
```

Comment:
That's horrific LOL! Can you change the two lines to... 

Line 42: ccd_case_number: payment?.ccd_case_number || ''
Line 46,47: customer_reference: payment?.customer_reference || ''